lib/rubocop/cop/style/if_with_semicolon.rb

```diff
@@ -17,26 +17,56 @@ class IfWithSemicolon < Base
         include OnNormalIfUnless
         extend AutoCorrector
 
-        MSG = 'Do not use if x; Use the ternary operator instead.'
+        MSG = 'Do not use if x; Use the ternary operator instead or if else structure.'
```

Comment:
It'd be nice to put code like `if x;` and `if/else` in backticks, so it'd be colored in the output. There should be an "an" before `if/else` and it's an expression, not a structure. :-)

Reply:
Btw, I also think it'd be nice to just a custom version of the message without the word `or` in it - after all when we analyze the code we know whether it should be a ternary op or an `if/elsif`.